saleor/graphql/schema.graphql

```diff
@@ -19124,6 +19142,25 @@ input CheckoutLineInput {
   price: PositiveDecimal
 }
 
+input CheckoutValidationRules {
+  """
+  The validation rules that can be applied to provided shipping address data.
+  """
+  shippingAddress: CheckoutAddressValidationRules
+}
+
+input CheckoutAddressValidationRules {
+  """
+  Determines if an error should be raised when the provided address doesn't have all required fields. Providing `country` code is always required.
+  """
+  checkRequiredFields: Boolean = true
+
+  """
+  Determines if an error should be raised when the provided address doesn'tmatch to expected format.
```

Comment:
"Determines if an error should be raised when the provided address doesn't match the format expected in a particular field." Can we also provide an example to illustrate what kind of format we mean here?